Rampastring.Tools.csproj

```diff
@@ -1,74 +1,40 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{9F40961F-535A-480C-93C5-6877ED6A8626}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Rampastring.Tools</RootNamespace>
+    <TargetFrameworks>net48;net6.0-windows10.0.22000.0</TargetFrameworks>
+    <TargetPlatformMinVersion>7.0</TargetPlatformMinVersion>
+    <UseWindowsForms>true</UseWindowsForms>
+    <Title></Title>
+    <Description>Rampastring's Generally Useful Library</Description>
+    <Company>Rampastring</Company>
+    <Product></Product>
+    <Copyright>Copyright © Rami 'Rampastring' Pasanen 2016-2021</Copyright>
+    <Trademark></Trademark>
+    <Version>1.1.0.0</Version>
+    <FileVersion>1.1.0.0</FileVersion>
+    <InformationalVersion>1.1.0.0</InformationalVersion>
```

Comment:
In `AssemblyInfo` I used wildcard versioning to let Visual Studio automatically generate the last two integers of the version number. Is the same doable here?